index.js

```diff
@@ -61,10 +70,10 @@ class APIClient {
         return response;
     }
 
-    async publish(target = 'default', token = this.fetchToken()) {
+    async publish({ target = 'default', deployPercentage, token = this.fetchToken() }) {
```

Comment:
The function should still work without having to pass `{}`

```suggestion
    async publish({ target = 'default', deployPercentage, token = this.fetchToken() } = {}) {
```